### PR TITLE
processesDetails GET endpoint

### PR DESCRIPTION
**Pull request by bottaio**
Making processesDetails more RESTful by changing POST endpoint into GET.

### CODE REVIEW

File: ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala [hunk 1]
```diff
@@ -90,14 +92,12 @@ class ProcessesResources(val processRepository: FetchingProcessRepository,
             }
           }
         } ~ path("processesDetails") {
```
Comment (lciolecki) [inline ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala]: Mayby better way is do GET /processes?names=[comma separeted names]? I think we should use only one endpoint with filter queries to fetching list of processes,

Comment (bottaio) [reply ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala]: /process endpoint serves different data than /processesDetails which returns more verbose representation of processes

Comment (lciolecki) [reply ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala]: Okey! First: better name for this endpoint is /processes-datails/. Second: where are tests? :)

File: ui/server/src/test/scala/pl/touk/nussknacker/ui/api/ProcessesResourcesSpec.scala [hunk 2]
```diff
@@ -42,21 +42,22 @@ class ProcessesResourcesSpec extends FunSuite with ScalatestRouteTest with Match
...
-  val routWithAdminPermission = withPermissions(processesRoute, testPermissionAdmin)
+  val routeWithAllPermissions = withAllPermissions(processesRoute)
+  val routeWithAdminPermission = withPermissions(processesRoute, testPermissionAdmin)
   val processActivityRouteWithAllPermission = withAllPermissions(processActivityRoute)
   implicit val loggedUser = LoggedUser("lu",  testCategory)
 
   private val processName = ProcessName(SampleProcess.process.id)
 
   test("return list of process") {
     saveProcess(processName, ProcessTestData.validProcess) {
```
[the remark this answers is not in the document]
Comment (bottaio) [reply ui/server/src/test/scala/pl/touk/nussknacker/ui/api/ProcessesResourcesSpec.scala]: I'll fix these tests in another PR as for now it blocks further development and I'm short on time

**Pull request merged**